docs/conf.py

```diff
@@ -0,0 +1,173 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+import os
+import sys
+from sphinx.util import logging
+logger = logging.getLogger(__name__)
+
+
+# To fix 'docs/contents.rst not found' errors we need this, see
+# https://github.com/readthedocs/readthedocs.org/issues/2569
+
+master_doc = 'index'
+
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+def get_scripts(directory):
+    """
+    Find all the scripts available
+    """
+    scripts = []
+
+    # Walk the tree.
+    for root, directories, files in os.walk(directory):
+        for filename in files:
+            if filename.endswith(".py") and "_init_" not in filename:
+                scripts.append(filename.replace('\n', '').replace(".py", ""))
+
+    return scripts
+
+
+def find_scripts_entry():
+    """
+    Find the entries corresponding to the scripts.
+    """
+
+    with open("scripts.rst") as file:
+        lines = file.readlines()
+
+    entries = []
+    for line in lines:
+        if "automodule" in line:
+            values = line.split(".. automodule::")
+            entries.append(values[1].replace('\n', '').strip())
+    return entries
+
+
+def compare(list1, list2):
+    """
+    Return the elements not common to both lists
+    """
+    return [i for i in list1 + list2 if i not in list1 or i not in list2]
+
+
+# List of directories to scan and add the path.
+directories = ['../omero/analysis_scripts', '../omero/export_scripts',
+               '../omero/figure_scripts', '../omero/import_scripts',
+               '../omero/util_scripts']
+scripts = []
+for d in directories:
+    sys.path.insert(0, d)
+    scripts.extend(get_scripts(d))
+
+entries = find_scripts_entry()
+
+# Indicate the scripts not listed for documentation
+if len(entries) < len(scripts):
+    common = compare(scripts, entries)
+    logger.warning("automodule entries missing for:\n" + '\n'.join(common))
+
+
+# -- Project information -----------------------------------------------------
+
+project = u'omero scripts'
+copyright = u'2021, Open Microscopy Environment'
+author = u'Open Microscopy Environment'
+
+# The full version, including alpha/beta/rc tags
+# The short X.Y version.
+version = '5.6.0'
```

Comment:
This will need to be kept in sync with https://github.com/ome/omero-scripts/blob/develop/setup.py#L76? Should we look into way to [single source it](https://packaging.python.org/guides/single-sourcing-package-version/#single-sourcing-the-version)

Reply:
I was thinking about that too. I am going to see if I can unify that work

Reply:
I have prepared things and added info in README
about the release process of that repository
I had to manually fix the version in ``conf.py`` when this is merged. ``bump2version`` will synchronize the version